README.md

```diff
@@ -62,6 +63,43 @@ AUTHENTICATION_BACKENDS = [
 
 Note that if the you want to check the local DB for if the user exists there first, choose either `EmailOrUsernameModelBackend` or `ModelBackend` based on which of these backends is used in the authorization server. Using `EmailOrUsernameModelBackend` in the authorization server but using `ModelBackend` in the client application will mean that any time a user logs in to the client app with their email, the authentication request will always be forwarded to the authorization server even if the user account already exists in the local DB.
 
+### OCSTokenAuthentication Backend
+
+If the client application is using Django Rest Framework and should support api token authentication, switch out use of rest frameworks TokenAuthentication with this backend which performs TokenAuthentication on the authtoken and then falls back on the api_token within the AuthProfile model. It can be included by updating the following in the settings:
+
+```python
+REST_FRAMEWORK = {
+    ...
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        # Allows authentication against DRF authtoken and then Oauth Servers api_token
+        'ocs_authentication.backends.OCSTokenAuthentication',
+        'rest_framework.authentication.SessionAuthentication',
+    ),
+     ...
+}
+```
+
+### IsServer Permission
+
+This permission is used to allow the oauth server to call views within other applications, using its `OAUTH_SERVER_KEY`. This key should be kept private and only known by the applications and Oauth server. This permission should be included as the permission class on any view you want only accessible by the Oauth server.
```

Comment:
maybe I'm reading this wrong, but are you sure it should be `it's` instead of `its` for "using its OAUTH_SERVER_KEY" sentence? Or if there a different `its`?

Reply:
You're right. My bad, ignore